.dockerignore

```diff
@@ -0,0 +1,4 @@
+.git
+**/.gitignore
+**/.DS_Store
+test.bash
```

Comment:
Not needed as we're keeping in base, but it does affect how much data is sent to the build server.